client/app/counter/counter.service.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError as observableThrowError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class CounterService {
+
+    constructor(
+        private http: HttpClient
+    ) { }
+
+    getCounter(): Observable<any> {
```

Comment:
nit: return types.

Reply:
not sure what you mean by this comment...